net/sslh/src/opnsense/mvc/app/controllers/OPNsense/Sslh/forms/settings.xml

```diff
@@ -0,0 +1,213 @@
+<form>
+    <field>
+        <id>settings.enabled</id>
+        <label>Enable SSLH</label>
+        <type>checkbox</type>
+        <help><![CDATA[
+            This will enable the SSLH service.
+            ]]>
+        </help>
+    </field>
+    <field>
+        <id>settings.listen_addresses</id>
+        <label>Listen Addresses</label>
+        <help><![CDATA[
+            <b>-p, --listen</b>  address:port<br>
+            Interface and port on which to listen, e.g. foobar:443, where
+            foobar is the name of an interface (typically the IP address on
+            which the Internet connection ends up).<br>
+            <br>
+            This can be specified several times to bind sslh to several
+            addresses.
+            ]]>
+        </help>
+        <style>tokenize</style>
+        <allownew>true</allownew>
+        <hint>hostname:443,hostname:1194</hint>
+        <type>select_multiple</type>
+    </field>
+    <field>
+        <id>settings.transparent</id>
+        <label>Transparent</label>
+        <help><![CDATA[
+            <b>-t, --transparent</b><br>
+            Makes sslh behave as a transparent proxy, i.e. the receiving
+            service sees the original client's IP address.  This works on Linux
+            only and involves iptables settings.  Refer to the README for more
+            information.<br>
+            <br>
+            Default: false
+            ]]>
```

Comment:
since you are on FreeBSD here, this cannot work. The firewall is pf, not iptables / nftables

Reply:
I'll remove it.